modules/015-admission-policy-engine/openapi/config-values.yaml

```diff
@@ -3,8 +3,19 @@ properties:
   podSecurityStandards:
     type: object
     default: {}
-    description: "Pod Security Standards policy settings."
+    description: "Pod Security Standards policy settings (PSS)."
     properties:
+      defaultProfile:
```

Comment:
Seems it have to be `defaultPolicy` here, to be synchronized with PSS labels

Reply:
done